apps/src/templates/sectionAssessments/FreeResponsesAssessments.jsx

```diff
@@ -0,0 +1,125 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
+
+export const COLUMNS = {
+  STUDENT: 0,
+  RESPONSE: 1,
+};
+
+const freeResponsesDataPropType = PropTypes.shape({
+  id:  PropTypes.number,
+  studentId: PropTypes.string,
+  name: PropTypes.string,
+  response: PropTypes.string,
+});
+
+class FreeResponsesAssessments extends Component {
+  static propTypes= {
+    freeResponses:PropTypes.arrayOf(freeResponsesDataPropType),
+  };
+
+  state = {
+    [COLUMNS.NAME]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = (selectedColumn) => {
+    this.setState({
+      sortingColumns: sort.byColumn({
+        sortingColumns: this.state.sortingColumns,
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc',
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  studentResponseColumnFormatter = (studentAnswers, {rowData, rowIndex, columnIndex}) => {
+    const studentResponse = this.props.freeResponses[rowIndex].response;
```

Comment:
i would think that you can access the response from data supplied by reactabular, rather than doing the extra work of grabbing the response out of props... is this true? (same goes for `studentNameColumnFormatter` on line